lib/last_login_sync/use_case/populate_user_last_login.rb

```diff
@@ -0,0 +1,18 @@
+require 'date'
+
+class LastLoginSync::UseCase::PopulateUserLastLogin
+  def initialize(active_users_gateway:, last_login_gateway:)
+    @active_users = active_users_gateway
+    @last_login_gateway = last_login_gateway
+  end
+
+  def execute(date: nil)
```

Comment:
date: Date.today as a default?

Reply:
:+1:
Wasn't sure when evaluation happens, but looks like it's at execution each time, instead of at definition time